src/content/en/2020/markup.md

```diff
@@ -352,13 +345,11 @@ Standard elements are those that are or were part of the HTML specification. Whi
 <figcaption>{{ figure_link(caption="Low probabilities of finding a given element in pages of the sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
 </figure>
 
-{# TODO(authors): Interpet results. #}
```

Comment:
Explanation: Suggesting to skip this, or ask whether @catalinred or @iandevlin like to draft something.

Reply:
Ditto

Reply:
Again, I’d look at @catalinred or @iandevlin whether they like to expand on this.

Other than that, as opposed to the lang section here I don’t agree. Unless you’re coming from an Almanac convention that requires every data point to be interpreted, I wouldn’t be convinced that data can’t stand by itself. I’d suggest that not only are our readers capable of drawing conclusions, but that this can even be refreshing from an editorial perspective, if not to suggest—“wait a minute; what’s this, what does this mean?”

If I misunderstand you, please let me know, otherwise I’d appreciate a bit of leeway here for us to decide on what also _not_ to interpret.

Reply:
My opinion is we should be selective to what we put in the chapter. We have a lot of data - some of it will be interesting and some not. I think we need to pick and choose what to include to keep the chapter interesting.

The point of the almanac is "The Web Almanac is a comprehensive report on the state of the web, backed by real data and trusted web experts" rather than "a list of stats, with experts interpreting them".

Saying that, I think these particular stats ARE interesting. Why are they not used? Are they old elements which are not useful? Have they been replaced by better alternatives? Or are they new elements that haven't taken off yet?

Looking at MDN two if them are obsoleted ([`dir`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dir) and [`basefont`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/basefont) and one ([`rb`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/rp)) is Ruby specific - to me there's a question if that should really be a standard HTML element? Ultimately, I'd never heard of these particular elements and had to look them up to figure this information out, so think a summary of this explaining this would be useful here to save other readers doing the same.

However, I do think this is the authors work, so if you still feel this is not necessary explanation and the stats stand on their own, then I can accept this. But think it's right for @rviscomi to at least ask these questions in case you hadn't considered that interpretation or were making assumptions as to what readers might know because you are HTML experts (and other readers might not be).

Reply:
Suggestion to skip SGTM. Could you update the md to remove the content?

Reply:
I actually felt like I want to add info here. Added a paragraph. What do you think?